src/playback/internal/playbackcontroller.cpp

```diff
@@ -213,7 +213,7 @@ void PlaybackController::seek(const midi::tick_t tick)
         return;
     }
 
-    seek(tickToSecs(tick));
+    seek(playedTickToSecs(tick));
```

Comment:
This method also looks very questionable; it's completely unclear whether it expects raw or played ticks. If you want, you could figure that out and include a fix in a separate commit, but that's absolutely not a must. Perhaps it's better to save that for a different PR.

Reply:
Would something like `seekPlayedTick` be fine? 

Also, I made `PlaybackController::seek` public again (it was made private in #23011) but I'm now realizing I could instead keep it private and make a new public function (something like `PlaybackController::seekBeat`) to be called by `NotationActionController::move`. Would that be better?

Reply:
I'm not per se against making `seek` public (it can already be accessed anyway via the more cumbersome way of calling `dispatcher()->dispatch("rewind", ………)`). But introducing a `seekBeat` method might be even better indeed. Perhaps it's nice to put it just below `seekElement`.

Reply:
> Would something like seekPlayedTick be fine?

Judging from how `m_currentTicks` is used, it looks like that is actually raw ticks. So I would assume this method is rather `seekRawTick`, and thus should use `rawTickToSecs`.

Reply:
Hm, wouldn't it be played ticks since, for example, `PlaybackController::seekElement` uses a value from `NotationPlayback::playPositionTickByElement`, which returns played ticks?

Reply:
I researched it a bit more, and found a bug which I logged as https://github.com/musescore/MuseScore/issues/23795. 

`m_currentTicks` is indeed a raw tick, but this function works with played ticks. However, it turns out that in all cases where this function is called, the value that will become the argument of this function is a raw tick, but it is then converted to a played tick before calling the function.

So, the solution would be to make this function receive raw ticks, and adjust the callers to remove the conversion to played ticks.

There should be exactly three callers: `seekElement`, `seekListSelection`, and `setTempoMultiplier`. 
In the current master branch, there is a fourth one though: `rewind`. It tries to pass milliseconds, instead of seconds or ticks. There is no overload for milliseconds, so this (wrongly) uses the ticks overload. However, you don't need to worry about this, because I fixed it in #23794.